### ISSUE TITLE
[Issue Report]: Touch controls overlap with in-game buttons

### ISSUE THREAD

**Issue by ProvokedGamer** [diasurgical/devilutionX#7336]
### Operating System

iOS

### DevilutionX version

1.5.2

### Describe

The touch controls (joystick and the other buttons) overlap with the in-game buttons for quest, spell, etc.

![A screenshot to show what I mean](https://github.com/user-attachments/assets/9d59b1ec-9edd-4095-9300-5dee1ad2dfe7)

### To Reproduce

1. Sideload the IPA on iPhone 8+
2. Open the app and start a game
3. You should see them overlap now

### Expected Behavior

The touch controls shouldn’t overlap and it should be easier to press the buttons that the touch controls are blocking.

### Additional context

iPhone 8+
iOS 15.6

**Comment by StephenCWills** [comment]
The size and position is based on your screen's dpi. Whether you have room away from the main panel to render the touch controls depends on how much clearance your aspect ratio gives you on either side of the main panel. Just about the only thing you can do about this is to increase the game resolution to make the main panel smaller.

If all you need to do is tap the controls on the main panel, then you can tap somewhere away from the touch controls and drag your finger over the main panel button. When you release, it should activate the button.

This won't be fixed until we manage to implement customizable GUI elements via Lua. Most likely, not until 1.7.0. I'm closing this as it is more or less a duplicate of #6992.

**Comment by AJenbo** [comment]
It's not intended that the Main panel should be interacted with for touch devices, it's only for visual feedback like health, mana, info box and XP bar. The plan from my side is that the main panel gets replaced with a much more minimalist one that works better for a touch style interface.

P.s. use the buttons in the top right for opening panels
